packages/plugin/webpack/src/Config.ts

```diff
@@ -48,6 +49,20 @@ export interface WebpackPluginEntryPoint {
    * for all entries.
    */
   nodeIntegration?: boolean;
+  /**
+   * Custom options to merge into the configuration passed to `HtmlWebpackPlugin`.
+   */
+  htmlOptions?: Partial<HtmlWebpackPlugin.Options>;
+  /**
+   * Plugins to include before `HtmlWebpackPlugin`; typically, HTML plugin add-ons will
+   * need to be placed here.
+   */
+  htmlPlugins?: WebpackPluginInstance[];
+  /**
+   * Additional options to merge into the Webpack `output` configuration for this entry-
+   * point.
+   */
+  output?: object;
```

Comment:
I'm not clear on the use case for this option, it directly contradicts the goals of forge which is to manage these disk assets (E.g. we don't allow configuration of out/dir for other tools) specifically so we're always in control of and know where things are on disk.  i.e. this will break the environment variables that we provide for folks to get the path to the webpack generated files

Can you clarify exactly what you want to pass in here and why?

Reply:
yes -- in some cases, plugins, need to adjust the `output` parameter in concert with a hook into the build. for example, the SRI plugin needs `{output: {crossorigin: 'anonymous'}}`, but only for targets which invoke `HtmlWebpackPlugin`.

i did what i could to avoid this object, but i don't see an alternative -- any ideas?